index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
```

Comment:
<html lang="en">
This information helps search engines return language specific results.